docs/products/clickhouse/howto/clickhouse-query-cache.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Use query cache in Aiven for ClickHouse®
+sidebar_label: Use query cache
+---
+
+Aiven for ClickHouse® provides a query cache mechanism that helps improve query performance
+by caching query results.
+
+## How it works
+
+When the Aiven for
+[ClickHouse query cache](https://clickhouse.com/docs/en/operations/query-cache) is enabled,
+multiple identical `SELECT` queries run simultaneously are computed only once,
+and further executions of the same query are served directly from the cache. If the result
+of a SELECT query changes, the database invalidates and discards cached query results.
+Operations that change the data are `INSERT`, `UPDATE`, or `DELETE`.
+
+:::important
+By default, the Aiven for ClickHouse query cache is disabled for all `SELECT` queries.
+:::
+
+## Why use it
+
+Using query cache in your Aiven for ClickHouse services can help reduce latency and
+resource consumption.
+
+Key use case for the Aiven for ClickHouse query cache are the following:
+
+- Performance enhancement: Reducing latency and load for frequently executed queries
+- Analytical workloads: Using complex repetitive queries that involve high data aggregation
+- Data pipelines: [Using materialized views](/docs/products/clickhouse/concepts/federated-queries)
+  to optimize recurring data transformation and aggregation
+
+## Enable query cache
+
+To enable the query cache for a query, use an SQL client (for example, the
+[ClickHouse client](/docs/products/clickhouse/howto/connect-with-clickhouse-cli)) and
+append `SETTINGS use_query_cache = 1` to your query, for example:
+
+```sql
+SELECT 1
+SETTINGS use_query_cache = 1;
+```
+
+## Configure query cache
+
+To configure the query cache settings, use an SQL client (for example, the
+[ClickHouse client](/docs/products/clickhouse/howto/connect-with-clickhouse-cli)) and
+append a defined setting to your query, for example:
+
+```sql
+SELECT 1
+SETTINGS use_query_cache = 1, query_cache_min_query_runs = 5000;
+```
+
+You can configure the following query cache settings:
+
+- `enable_writes_to_query_cache`
+- `enable_reads_from_query_cache`
+- `query_cache_max_entries`
+- `query_cache_min_query_runs`
+- `query_cache_min_query_duration`
+- `query_cache_compress_entries`
+- `query_cache_squash_partial_results`
+- `query_cache_ttl`
+- `query_cache_share_between_users`
+
+## Limitation
+
+- Maximum query cache size: 64 MiB for each GiB of RAM (for example, 256 MiB for a 4-GiB
+  service or 1 GiB for a 16-GiB service)
+- Maximum number of query cache entries: 64 entries for each GiB of RAM (for example,
+  1024 entries for a 16-GiB service)
+- Query cache is not long-lived: cached results are invalidated when the underlying data
```

Comment:
No, if data changes, the cached result is not invalidated and may return incorrect result https://clickhouse.com/blog/introduction-to-the-clickhouse-query-cache-and-design